rust/stackable-cockpit/src/platform/manifests.rs

```diff
@@ -74,6 +74,11 @@ pub trait InstallManifestsExt {
     ) -> Result<(), Error> {
         debug!("Installing demo / stack manifests");
 
+        let mut parameters = parameters.clone();
+        // We add the NAMESPACE parameter, so that stacks/demos can use that to render e.g. the
+        // fqdn service names [which contain the namespace].
+        parameters.insert("NAMESPACE".to_owned(), product_namespace.to_owned());
```

Comment:
note: I would add the parameter at the previous step (where `install:manifests` is called). This would only require a clone of the product namespace instead of the complete hash map.

note: I would use `productNamespace` as the templating variable instead of `NAMESPACE`.

Reply:
I intentionally added it to `install_manifests`, as it is called in `StackSpec::install_manifests` as well as `DemoSpec::install_manifests`.
This way we avoid duplicated code and logic.
I think the clone is acceptable im comparison to code complexity/maintainability.

Regarding the naming we have https://github.com/stackabletech/stackable-cockpit/pull/355#discussion_r1978813195, I will update the PR accordingly.

Reply:
Alright, let's keep the implementation as is. The naming will discussed in https://github.com/stackabletech/stackable-cockpit/pull/355#discussion_r1973171123.